extras/coverage-report.sh

```diff
@@ -1,7 +1,18 @@
-# Run tests, generate coverage report and open it on a browser
+#!/bin/sh
+# Run tests, generate coverage report and open it in a browser
 #
 # Requires: coverage 3.3 or above from http://pypi.python.org/pypi/coverage
 
-coverage run --branch $(which trial) --reporter=text tests
-coverage html -i
+trial="`which trial`"
```

Comment:
Well, the problem with coverage is that many Scrapy tests use subprocess to run Scrapy commands, so coverage results won't be precise. I don't know if anyone is using this script. Also, I don't know much about our debian package, so it is better to ask other developers :) My vote is to remove this script.

To run coverage in tox add `pytest-cov` and `coverage` to tests/requirements.txt, then run e.g. `tox -e py27 -- --cov scrapy tests/test_proxy_connect.py`